### PR TITLE
chore: invoke deprecated methods in default supports method for backwards compatibility

### PR DESCRIPTION
**Pull request by filipelautert**
### TLDR: 
Invoke deprecated methods in default supports method to ensure that extensions are not broken, but one day when those are removed it will return only true

### Description
The new support `method` was introduced in PR https://github.com/liquibase/liquibase/pull/5619 so specially NoSQL databases can check for other supports as `Collections` without changing Database class and adding a `supportsCollections()` method.
In core all the locations were `supportsSequences()` existed were changed to supports(Sequence) , and all *Database classes had this new method implemented. But extensions don't have it, so they could return the wrong value. For instance, BigQuery doesn't support sequences but would start to return true to core.
With this fix we continue to return what is expected.

## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [X] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by StevenMassaro — APPROVED** [review]
Code here looks fine but I don't know any of the context around this change.

**Comment by filipelautert** [comment]
> Code here looks fine but I don't know any of the context around this change.

@StevenMassaro the new support `method` was introduced so specially NoSQL databases can check for other supports as `Collections` without changing Database class and adding a `supportsCollections()` method.
In core all the locations were `supportsSequences()` existed were changed to supports(Sequence) , and all *Database classes had this new method implemented. But extensions don't have it, so they could return the wrong value. For instance, BigQuery doesn't support sequences but would start to return true to core.
With this fix we continue to return what is expected.

I'm adding this info to the PR description, thanks!

**Review by MalloD12 — APPROVED** [review]
LGTM
